src/main/java/uk/gov/hmcts/divorce/divorcecase/validation/ValidationUtil.java

```diff
@@ -156,6 +159,16 @@ public static List<String> validateCaseFieldsForIssueApplication(MarriageDetails
         );
     }
 
+    public static List<String> validateCitizenResendInvite(CaseDetails<CaseData, State> details) {
+        var data = details.getData();
+        boolean isApplicant2EmailUpdatePossible = details.getState() == State.AwaitingApplicant2Response
+            && data.getCaseInvite().accessCode() != null
+            && data.getApplicationType() == ApplicationType.JOINT_APPLICATION;
+        return isApplicant2EmailUpdatePossible
+            ? emptyList()
+            : singletonList("Not possible to update applicant 2 invite email address");
```

Comment:
Will this message be shown on the UI if yes worth checking with BA's

Reply:
The event will only be for citizens so the error message will just be read in logs. Is that ok?

Reply:
ok if it is in only in logs that should be fine thanks